Sources/Redis/Client/RedisClient.swift

```diff
@@ -21,9 +21,6 @@ public final class RedisClient: DatabaseConnection, BasicWorker {
     /// The channel
     private let channel: Channel
 
-    /// Currently executing `send(...)` promise.
-    private var currentSend: Promise<Void>?
```

Comment:
Storing the currentSend is important for failing the request in the case of a network error. Maybe instead of storing just a single promise we can store an array here?